File: C#/begginer/1012.cs
using System; 

class URI {

    static void Main(string[] args) { 

        string[] input = Console.ReadLine().Split(' ');

        double a = double.Parse(input[0]), b = double.Parse(input[1]), c = double.Parse(input[2]);

        Console.WriteLine($"TRIANGULO: {(a * c) / 2:F3}");
        Console.WriteLine($"CIRCULO: {3.14159 * Math.Pow(c, 2):F3}");
        Console.WriteLine($"TRAPEZIO: {((a + b) * c) / 2:F3}");
        Console.WriteLine($"QUADRADO: {Math.Pow(b, 2):F3}");
        Console.WriteLine($"RETANGULO: {a * b:F3}");

    }

}

File: Java/begginer/1012.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
 
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        
        String[] input = scanner.nextLine().split(" ");
        
        double a = Double.valueOf(input[0]).doubleValue();
        double b = Double.valueOf(input[1]).doubleValue();
        double c = Double.valueOf(input[2]).doubleValue();
        
        System.out.printf("TRIANGULO: %.3f\n", a * c / 2);
        System.out.printf("CIRCULO: %.3f\n", 3.14159 * Math.pow(c, 2));
        System.out.printf("TRAPEZIO: %.3f\n", (a + b) * c / 2);
        System.out.printf("QUADRADO: %.3f\n", Math.pow(b, 2));
        System.out.printf("RETANGULO: %.3f\n", a * b);
    }
 
}
